rename_deprecated_modules.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+import logging
+
+from openupgradelib import openupgrade
+
+_logger = logging.getLogger(__name__)
+
+env = env  # noqa F821 undefined name 'env'
```

Comment:
No need to edit this now, but this won't be parsed correctly. The correct syntax is `# noqa: F821`.